### PR TITLE
More on resilience and SR practices

### PR DESCRIPTION
**Pull request by ivorc**

### CODE REVIEW

File: practices/service-reliability.md [hunk 1]
```diff
@@ -7,13 +7,21 @@
 
 ## Details
 
+* Understand reliability requirements.
+    * Determine [service level indicators and objectives](https://landing.google.com/sre/sre-book/chapters/service-level-objectives/) (SLIs and SLOs) to represent these requirements.
```
Comment (andyblundell) [inline practices/service-reliability.md:11]: Still worried this one may not be aligned with NHSD Service Management taxonomy - can we make this slightly more generic?

File: practices/service-reliability.md [hunk 2]
```diff
@@ -7,13 +7,21 @@
...
+* Plan for failure.
+    * Rehearse incident classification and resolution using regular [Game Days](https://aws.amazon.com/gameday/#:~:text=What%20is%20AWS%20GameDay%3F,gamified%2C%20risk%2Dfree%20environment.) and similar exercises.
+    * Remember that service _reliability_ encompasses security, data integrity, performance, capacity and other factors as well as availability.
 * Design for failure: assume everything will fail at some point, so:
     * Implement loose coupling where possible to minimise the impact of component failure.
-    * Infrastructure should all be appropriately resilient, for example via tools such as [AWS auto-scaling groups](https://docs.aws.amazon.com/autoscaling/ec2/userguide/AutoScalingGroup.html).
-* Services should scale automatically up and down.
+    * Infrastructure should all be appropriately resilient (see [cloud services](cloud-services.md)).
```
Comment (andyblundell) [inline practices/service-reliability.md:19]: Let's add a link in the other direction (i.e. from cloud-services to service-reliability)

File: practices/service-reliability.md [hunk 3]
```diff
@@ -7,13 +7,21 @@
 
 ## Details
 
+* Understand reliability requirements.
+    * Determine [service level indicators and objectives](https://landing.google.com/sre/sre-book/chapters/service-level-objectives/) (SLIs and SLOs) to represent these requirements.
+    * Use SLIs and SLOs to drive monitoring and alerting requirements. Consider what constitutes [meaningful availability](https://www.usenix.org/system/files/nsdi20spring_hauer_prepub.pdf) for your service.
+    * Agree an incident severity classification and the response which each level requires in terms of communication and resolution (example: [HSCN Consumer Handbook](https://digital.nhs.uk/services/health-and-social-care-network/hscn-consumer-handbook/service-levels-and-incident-severity-classification)). Be pragmatic and don't overstate the requirements: targeting unnecessary levels of resilience is wasteful and reduces the team's ability to deliver functionality.
```
[the remark this answers is not in the document]
Comment (ivorc) [reply practices/service-reliability.md]: Addressed.

File: practices/service-reliability.md (hunk 1 above)
Comment (ivorc) [reply practices/service-reliability.md]: Addressed.

File: practices/service-reliability.md (hunk 2 above)
Comment (ivorc) [reply practices/service-reliability.md:18]: Addressed.

**Review by andyblundell — APPROVED** [review]

**Pull request merged**